madgraph/interface/madgraph_interface.py

```diff
@@ -674,11 +674,11 @@ def help_add(self):
         logger.info("   --output=  : Specify the name of the directory where the merge is done.")
         logger.info("                This allow to do \"import NAME\" to load that merge.")
         logger.info("   --recreate : Force to recreated the merge model even if the merge model directory already exists.")
-        
+
     def help_convert(self):
         logger.info("syntax: convert model FULLPATH")
-        logger.info("modify (in place) the UFO model to make it compatible with both python2 and python3")
-        
+        logger.info("modify (in place) the UFO model to make it compatible with python3")
```

Comment:
> They are just one change (on the help/comment of the convert model that I would not include).

@oliviermattelaer you're just saying that this change should be reverted, correct? 

If so, I'll get this tomorrow and then also clean up the commits a bit and squash them into 1 to make it cleaner.

It looks like the removal of all use of `six` is probably going to be big enough that I should do that in another PR that only touches that.

Beyond using a Docker container that has all the other dependencies setup in it, is there a way to run the tests suite

https://github.com/mg5amcnlo/mg5amcnlo/blob/57eea3e92d06d2fb27cd3c0e66ced15ed2d05a44/README.developer#L33-L43

automatically?